app/view/settings/SeatView.js

```diff
@@ -963,10 +963,23 @@ SDL.SeatView = Em.ContainerView.create({
             content: 'ID'
         }),
 
-        seatModuleUUID: SDL.Label.create({
+        seatModuleUUID: Em.Select.create({
             elementId: 'seatModelCurrent',
             classNames: 'seatModelCurrent',
-            contentBinding: 'SDL.RCModulesController.currentSeatModuleID'
+            change: function() {
+              var desiredUUID = this.selection;
+              var location_name = '';
+              for(var key in SDL.RCModulesController.seatModels) {
+                if(desiredUUID === SDL.RCModulesController.seatModels[key].UUID) {
+                  location_name = SDL.RCModulesController.seatModels[key].ID;
+                  break;
+                }
+              }
+              if(location_name === '') {
```

Comment:
@YarikMamykin 
```suggestion
              if(location_name != '') {
```

Reply:
@AKalinich-Luxoft fixed in https://github.com/LuxoftSDL/sdl_hmi/pull/13/commits/093a2a9e3b833be5ec8227ea0cb0f1ddfe761f83